File: src/data/ListSingleDay.java
package data;

import java.util.Date;

public class ListSingleDay extends ListDay {

  private Date date;

  public ListSingleDay(Appointment[] appointments, Therapist[] therapists, Date date) {
    super(appointments, therapists);
    this.date = date;
  }

  public Date getDate() {
    return date;
  }

  public void setDate(Date date) {
    this.date = date;
  }

}

File: src/class/ListSingleDay.ts
import ListDay from './ListDay';
import SingleAppointment from './SingleAppointment';

export default class ListSingleDay extends ListDay {
  date : Date;

  constructor(
    appointments : SingleAppointment[],
    date: Date,
  ) {
    super(appointments);
    this.date = date;
  }
}
